chia/util/streamable.py

```diff
@@ -123,26 +151,26 @@ def convert_optional(convert_func: ConvertFunctionType, item: Any) -> Any:
 
 
 def convert_tuple(convert_funcs: List[ConvertFunctionType], items: Collection[Any]) -> Tuple[Any, ...]:
-    if len(items) != len(convert_funcs):
-        raise ValueError(f"Invalid size. Expected: {len(convert_funcs)}, got: {len(items)}")
     if not isinstance(items, (list, tuple)):
-        raise TypeError(f"expected: tuple or list, actual: {type(items).__name__}")
+        raise InvalidTypeError(tuple, type(items))
```

Comment:
You know I like my passing by name...  Enough to mention it, not enough to raise a concern over it here.  Though the arguments could probably just be `expected` and `actual` since based on the exception type the `_type` or `_size` etc suffixes can be deduced.